src/super_gradients/training/datasets/detection_datasets/detection_dataset.py

```diff
@@ -457,6 +458,45 @@ def _get_cached_image(self, index: int, cached_image_shape: Tuple[int, int]) ->
         resized_image = padded_image[:cached_height, :cached_width, :]
         return resized_image.copy()
 
+    def __convert_input_dict_to_detection_sample(self, sample_annotations: Dict[str, Union[np.ndarray, Any]]) -> DetectionSample:
+        bboxes_xyxy = sample_annotations["target"][..., :4]
+        labels = sample_annotations["target"][..., 4]
+        is_crowd = np.zeros_like(labels, dtype=bool)
+        if "crowd_target" in sample_annotations:
+            crowd_bboxes_xyxy = sample_annotations["crowd_target"][..., :4]
+            crowd_labels = sample_annotations["crowd_target"][..., 4]
+            bboxes_xyxy = np.concatenate([bboxes_xyxy, crowd_bboxes_xyxy], axis=0)
+            labels = np.concatenate([labels, crowd_labels], axis=0)
+            is_crowd = np.concatenate([is_crowd, np.ones_like(crowd_labels, dtype=bool)], axis=0)
+
+        bboxes_xywh = xyxy_to_xywh(bboxes_xyxy, image_shape=None)
+        return DetectionSample(
+            image=sample_annotations["image"],
+            bboxes_xywh=bboxes_xywh,
+            labels=labels,
+            is_crowd=is_crowd,
+            additional_samples=None,
+        )
+
+    def __convert_detection_sample_to_dict(self, detection_sample: DetectionSample, include_crowd_target: bool) -> Dict[str, Union[np.ndarray, Any]]:
+        image = detection_sample.image
+        crowd_mask = detection_sample.is_crowd > 0
+        crowd_labels = detection_sample.labels[crowd_mask]
+        crowd_bboxes_xyxy = detection_sample.bboxes_xyxy[crowd_mask]
+        crowd_target = np.concatenate([crowd_bboxes_xyxy, crowd_labels[..., None]], axis=-1)
```

Comment:
This is gonna crash when we there are no crowd bboxes since -
np.concatenate([np.array([]) ,x[...,None]]) have different dims.

Reply:
The example above should not happen, since we guaranteed that empty boxes has always shape of (0,4) so concat would work in this case.